classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_reader :student
```

Comment:
- [ ] Kindly make sure you are using the correct `typo` for your variables.
```suggestion
  attr_reader :students
```